python/paddle/fluid/regularizer.py

```diff
@@ -231,6 +263,95 @@ def __str__(self):
         return "L1Decay, regularization_coeff=%f" % self._regularization_coeff
 
 
+class WeightDecayBase(object):
+    def __str__(self):
```

Comment:
`__str__` is no need to be an abstract method, since all Python object has a `__str__` method.
It is no need to declare in base class